### ISSUE TITLE
 Mode Optimizations by Combination or Mixture of Features

### ISSUE THREAD

**Issue by CodexLink** [CodexLink/PortESTL-PerEMNFAD#12]
It feels I need to sort this out as well, All those modes that I have added is technically not finalized to the point I have to think of a something else to occupy it. But in this case, we have to case this one out of the problem before we think of any modes that could fit on this project.

The battle of these three modes could lead to a lot of things. One of them is a Combination or Mixture of modes.

So, in this case, we could do something such as

On [1], Instead of Status Modes @ (0,0), Add TimeClock as Mix of Digital Clock and Sensor Sensing Mode

On [2] is Just a WiFiDeauther Window. It should display the following with respective Y Axis
(0,0) - Mode Status of WiFiDeauther
(0,1) - SSID Connected
(0,2) - More Info of Mode
(0,3) - Additional for (0,2) with Arrow Indications

On [3] NodeMCU Status
Shows
(0,0) - Mode (Entry Mode, Independent Mode, Error, Unknown)
(0,1) - SSID Connected or If Dependent add SSID of NodeMCU
(0,2) - Password and IP
(0,3) - Serial Bytes and Client Count

On [4] Python Windows Integration Status
(0,0) - Mode (osu! Client, Technical Info)
For Technical Info
(0,0) - Bytes of Transfer Rate
(0,0) - Computer Lock Off Status and RFID Status
(0,0) - (We don't know)

Basically, at this point, I just finished this issue after creating it in just minutes.

**Comment by CodexLink** [comment]
This was now finalized. LOL XD
